juju/model.py

```diff
@@ -930,9 +966,7 @@ def debug_log(
 
         """
         if to:
-            placement = [
-                client.Placement(**p) for p in to
-            ]
+            placement = parse_placement(to)
```

Comment:
This makes a lot of sense. Thank you.

You can technically get rid of the call to parse_placement in application.py (though the "backwards compatibility" line in placement.py means that the multiple calls won't break anything).